File: src/com/bean/Worker.java
package com.bean;

public class Worker {
private int workercount;
private int sal;
public Worker(int workercount, int sal) {
	this.workercount = workercount;
	this.sal = sal;
}
public int getWorkercount() {
	return workercount;
}
public void setWorkercount(int workercount) {
	this.workercount = workercount;
}
public int getSal() {
	return sal;
}
public void setSal(int sal) {
	this.sal = sal;
}
@Override
public String toString() {
	return "Worker [workercount=" + workercount + ", sal=" + sal + "]";
}

}

File: src/com/bean/Worker.go
package bean

import (
	"fmt"
)

type Worker struct {
	workercount int
	sal         int
}

func CreateWorker(workercount int, sal int) Worker {
	return Worker{workercount: workercount, sal: sal}
}
func (w Worker) getWorkercount() int {
	return w.workercount
}
func (w Worker) setWorkercount(workercount int) {
	w.workercount = workercount
}
func (w Worker) getSal() int {
	return w.sal
}
func (w Worker) setSal(sal int) {
	w.sal = sal
}
func (w Worker) String() string {
	return fmt.Sprintf(" Worker [workercount =%d, sal=%d]", w.workercount, w.sal)
}
